webapp/src/RuntimeConfigurationProvider.tsx

```diff
@@ -26,7 +26,10 @@ interface RuntimeConfigurationProviderProps {
  */
 export const loadRuntimeConfiguration =
     async (): Promise<RuntimeConfiguration> => {
-        const config_file = import.meta.env.VITE_ENV == 'LOCAL' ? '/config.json' : '/config-prod.json'
+        const config_file =
+            import.meta.env.VITE_ENV == 'LOCAL'
```

Comment:
do we need in `.env.properties` duplicated
` VITE_ENV == 'LOCAL' ` ?
I was expecting Prod or similar